providers/fab/provider.yaml

```diff
@@ -35,6 +35,7 @@ source-date-epoch: 1747132761
 # In such case adding >= NEW_VERSION and bumping to NEW_VERSION in a provider have
 # to be done in the same PR
 versions:
+  - 2.2.0
```

Comment:
Remove this entry.
It causes issue with release procedure

Reply:
version_added is the next feature release 2.2.0 that is OK.

I specify the version number during release peocess.

Reply:
I won’t be able to point to 2.2.0 in version added if the versions does not specify 2.2.0 I think.

Last time in a similar setup I had documentation build error. But maybe that was caused by something else, I’ll try again.

Reply:
I dont think we cross refrence it. Some settings dont have version_added at all.

The only case we must specify future version in versions is when you wish to set it as min version for another package

Reply:
> The only case we must specify future version in versions is when you wish to set it as min version for another package

I think that was the intention here (airflow has fab provider >=2.2.0). And we need to do it if we want defaults to work - this is a one-time (or actually continuation of the previous) cleanups and I think it's worth doing it. The problem is that currently the old provider will not get the right default value when it is moved to FAB section and Airflow 3.0.2  will install older provider version. Those are the teething problems we need to solve now I think by cleaning the configs, otherwise it might cause the problem for our users.

And setting the right provider version now is the only way - I think.

Reply:
Indeed I forgot about that part but that was the original intent, continuation of the previous cleanup.

Reply:
OK then all is good